src/main/java/engine/config/reader/ConfigReader.java

```diff
@@ -0,0 +1,27 @@
+package engine.config.reader;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConfigReader {
```

Comment:
Maybe rename that to PropertyHelper, or PropertyManager? Since we are both reading and setting the properties here?

Reply:
@Fishgame1 This will be renamed as part of the singleton implementation task.